Makefile

```diff
@@ -0,0 +1,3 @@
+PHONY: test
+pytest test:
+	pytest --cov=./
```

Comment:
is py.test same as pytest?

Reply:
there is no py.test, only pytest. This syntax allows one to enter either `make pytest` or `make test` commands to run pytest with coverageg